playbooks/resize_disk.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- hosts: all
+  become: true
+  tasks:
+    - name: Find root partition
+      shell: df --output=source,fstype / | tail -n 1
+      register: root_partition
+
+    - name: Install cloud-utils-growpart package
+      package:
+        name: cloud-utils-growpart
+      when: ansible_os_family == 'RedHat'
+
+    - name: Install cloud-guest-utils package
+      package:
+        name: cloud-guest-utils
+      when: ansible_os_family == 'Debian'
+
+    - name: Install e2fsprogs package
+      package:
+        name: e2fsprogs
+      when: '"ext4" in root_partition.stdout'
+
+    - name: Extend partition
+      command: growpart /dev/vda {{ root_partition.stdout.split(' ')[0].replace('/dev/vda', '') }}
```

Comment:
I am wondering if this is always `vda`. `vda` is naming for virtio but on virtualbox it may fail if they use `sda`. Not sure if we have those, but worth considering.

Reply:
Can cross that bridge if someone gets there.